ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -95,7 +93,9 @@ final class StandaloneApiServer(
       )
       ledgerConfiguration = ledgerConfig.copy(
         // TODO: Remove the initial ledger config from readService.getLedgerInitialConditions()
-        initialConfiguration = initialConditions.config,
+        initialConfiguration = initialConditions.config.copy(
+          generation = initialConditions.config.generation + 1
```

Comment:
Should we maybe set the generation to 1 in the default ledger configurations?

Reply:
And completely disregard the value returned by `ReadService.getLedgerInitialConditions()`?